### ISSUE TITLE
Continous Integration on Action: for Python 3.10.alpha builds: AttributeError: module 'sysconfig' has no attribute 'get_preferred_scheme'

### ISSUE THREAD

**Issue by SylvainDe** [SylvainDe/DidYouMean-Python#57]
**First failure**

Date: 26th July
Build: https://github.com/SylvainDe/DidYouMean-Python/actions/runs/1065669199

```

Run pip install --upgrade pep8
ERROR: Exception:
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.10.0-alpha.1/x64/lib/python3.10/site-packages/pip/_internal/cli/base_command.py", line 173, in _main
    status = self.run(options, args)
  File "/opt/hostedtoolcache/Python/3.10.0-alpha.1/x64/lib/python3.10/site-packages/pip/_internal/cli/req_command.py", line 203, in wrapper
    return func(self, options, args)
  File "/opt/hostedtoolcache/Python/3.10.0-alpha.1/x64/lib/python3.10/site-packages/pip/_internal/commands/install.py", line 239, in run
    options.use_user_site = decide_user_install(
  File "/opt/hostedtoolcache/Python/3.10.0-alpha.1/x64/lib/python3.10/site-packages/pip/_internal/commands/install.py", line 652, in decide_user_install
    if site_packages_writable(root=root_path, isolated=isolated_mode):
  File "/opt/hostedtoolcache/Python/3.10.0-alpha.1/x64/lib/python3.10/site-packages/pip/_internal/commands/install.py", line 598, in site_packages_writable
    get_lib_location_guesses(root=root, isolated=isolated))
  File "/opt/hostedtoolcache/Python/3.10.0-alpha.1/x64/lib/python3.10/site-packages/pip/_internal/commands/install.py", line 584, in get_lib_location_guesses
    scheme = get_scheme(
  File "/opt/hostedtoolcache/Python/3.10.0-alpha.1/x64/lib/python3.10/site-packages/pip/_internal/locations/__init__.py", line 102, in get_scheme
    new = _sysconfig.get_scheme(
  File "/opt/hostedtoolcache/Python/3.10.0-alpha.1/x64/lib/python3.10/site-packages/pip/_internal/locations/_sysconfig.py", line 131, in get_scheme
    scheme_name = _infer_prefix()
  File "/opt/hostedtoolcache/Python/3.10.0-alpha.1/x64/lib/python3.10/site-packages/pip/_internal/locations/_sysconfig.py", line 45, in _infer_prefix
    return sysconfig.get_preferred_scheme("prefix")  # type: ignore
AttributeError: module 'sysconfig' has no attribute 'get_preferred_scheme'
Error: Process completed with exit code 2.
```

**Last success**

Date: 25th July
Build: https://github.com/SylvainDe/DidYouMean-Python/actions/runs/1063511128

Error is probably linked to changes in pip. A PR has been opened: 

https://github.com/pypa/pip/pull/10252

**Comment by SylvainDe** [comment]
Issue opened: https://github.com/pypa/pip/issues/10253 .

**Comment by SylvainDe** [comment]
Successfully fixed when CI went from using pip 21.2.2 to pip 21.2.3.
